server/enterprise/metrics/metrics.go

```diff
@@ -1503,6 +1541,44 @@ func (mi *MetricsInterfaceImpl) IncrementClusterEventType(eventType model.Cluste
 		mi.ClusterEventTypeInvSessions.Inc()
 	case model.ClusterEventInvalidateCacheForRoles:
 		mi.ClusterEventTypeInvRoles.Inc()
+	case model.ClusterEventInvalidateCacheForChannelGuestCount:
```

Comment:
Wondering if we could consider storing these in a map to make it easier to keep everything up to date and avoid missing more in the future.

Reply:
You mean a map in the `model` package itself?

Reply:
I guess it could live on either side, but having it on the model side may make it a little easier, as new additions would start there. A simple test could then ensure we are covering all supported caches.